packages/neuron-wallet/tests/block-sync-render/indexer-connector.test.ts

```diff
@@ -29,7 +29,7 @@ const connectIndexer = async (indexerConnector: IndexerConnector) => {
 
 describe('unit tests for IndexerConnector', () => {
   const nodeUrl = 'http://nodeurl:8114'
-  const indexerFolderPath = '/indexer/data/path'
+  const indexerFolderPath = process.platform === 'win32' ? '\\indexer\\data\\path' : '/indexer/data/path'
```

Comment:
I'd like to know why not `path.join` here?

Reply:
Good suggestion. I will update it using `path.join`. 👍